test/browser/routes.test.jsx

```diff
@@ -0,0 +1,29 @@
+var should = require('should');
+var React =require('react/addons');
+var TestUtils = React.addons.TestUtils;
+var Router = require('react-router');
+var Link = Router.Link;
+
+var routes = require('../../lib/routes.jsx');
+
+describe('routes', function() {
+  routes.URLS.forEach(function(url) {
+    describe(url, function() {
+      it('should contain 0 non-named links', function(done) {
```

Comment:
This test ensures we never regress back to using URL paths in the `to` property of `<Link>` components.